src/uct/tcp/tcp_iface.c

```diff
@@ -32,9 +32,13 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
    ucs_offsetof(uct_tcp_iface_config_t, sockopt_nodelay), UCS_CONFIG_TYPE_BOOL},
 
   {"SNDBUF", "64k",
-   "Socket send buffer size.",
+   "Socket send buffer size",
    ucs_offsetof(uct_tcp_iface_config_t, sockopt_sndbuf), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"RCVBUF", "64k",
```

Comment:
is it better than default?

Reply:
I don't see any performance changes om microbenchmarks. But it requires more performance studies. Also, it would be better to study it with a single connection per direction when it will be implemented in TCP.
I returned back to "auto".